scripts/lib/spreadsheet-mappings.ts

```diff
@@ -36,61 +41,93 @@ export const FIELD_MAPPINGS: { [index: string]: string[] } = {
+    multiple: ['Multiple'],
+    other: ['Other'],
+    unknown: ['Unknown'],
+    financing: ['Financing'],
+  },
+  type: {
+    rebate: ['Rebate (post purchase)'],
+    pos_rebate: ['Point of sale rebate'],
+    account_credit: ['Account Credit'],
+    tax_credit: ['Tax Credit'],
+    assistance_program: ['Assistance program'],
+    bonus: ['Bonus'],
+    multiple: ['Multiple'],
+    other: ['Other'],
+    unknown: ['Unknown'],
+    financing: ['Financing'],
   },
   item: {
-    'Ground Source Heat Pump (GSHP) / Geothermal HP':
-      'geothermal_heating_installation',
-    'HVAC - Air Source Heat Pump': 'heat_pump_air_conditioner_heater',
-    'HVAC - Air to Water Heat Pump': 'heat_pump_air_conditioner_heater',
-    'HVAC - Ducted Heat Pump': 'heat_pump_air_conditioner_heater',
-    'HVAC - Ductless Heat Pump': 'heat_pump_air_conditioner_heater',
-    'Heat Pump Water Heater (HPWH)': 'heat_pump_water_heater',
-    'New Electric Vehicle': 'new_electric_vehicle',
-    'Used Electric Vehicle': 'used_electric_vehicle',
-    'Electric Vehicle Charger': 'electric_vehicle_charger',
-    'Rooftop Solar': 'rooftop_solar_installation',
-    'Battery Storage': 'battery_storage_installation',
-    'Heat Pump Dryers / Clothes Dryer': 'heat_pump_clothes_dryer',
-    'Heat Pump Dryers / clothes dryer': 'heat_pump_clothes_dryer',
-    'Electric Stove': 'electric_stove',
-    'Weatherization (insulation and air sealing)': 'weatherization',
-    'Electric wiring': 'electric_panel',
-    'Electric panel': 'electric_panel',
-    'Electric lawn equipment (mower, edger, leaf blower, weedwhacker)':
-      'electric_outdoor_equipment',
-    'Smart Thermostat': 'smart_thermostat',
-    'E-Bike': 'ebike',
-    'Induction Cooktop': 'electric_stove',
-    'Electric Thermal Storage/Slab': 'electric_thermal_storage_and_slab',
-    'Evaporative Cooler': 'evaporative_cooler',
-    'Non-Heat Pump Clothes Dryer': 'non_heat_pump_clothes_dryer',
-    'Non-Heat Pump Water Heater': 'non_heat_pump_water_heater',
-    'Whole House Fan': 'whole_house_fan',
-    Other: 'other',
+    geothermal_heating_installation: [
+      'Ground Source Heat Pump (GSHP) / Geothermal HP',
+    ],
+    heat_pump_air_conditioner_heater: [
+      'HVAC - Air Source Heat Pump',
+      'HVAC - Air to Water Heat Pump',
+      'HVAC - Ducted Heat Pump',
+      'HVAC - Ductless Heat Pump',
+    ],
+    heat_pump_water_heater: ['Heat Pump Water Heater (HPWH)'],
+    new_electric_vehicle: ['New Electric Vehicle'],
+    used_electric_vehicle: ['Used Electric Vehicle'],
+    electric_vehicle_charger: ['Electric Vehicle Charger'],
+    rooftop_solar_installation: ['Rooftop Solar'],
+    battery_storage_installation: ['Battery Storage'],
+    heat_pump_clothes_dryer: ['Heat Pump Dryers / Clothes Dryer'],
+    electric_stove: ['Induction Cooktop', 'Electric Stove'],
+    weatherization: ['Weatherization (insulation and air sealing)'],
+    electric_panel: ['Electric panel', 'Electric wiring'],
```

Comment:
capitalization doesn't matter for these in the end, right? Observing that some columns are all words capitalized, some have only the first letter in the column, some none at all.

If it doesn't matter, should we be consistent throughout - maybe only lowercase?

Reply:
My current thinking is that the first alias at least is going to be special – when the back-to-csv story is done, that first alias will be treated as the "canonical alias" – that is, the user-friendly string that we allow folks to use when filling out the spreadsheets so they can work with English phrases instead of the machine-friendly stuff.

We could separate those concepts (so that there is a separate "canonical alias" data structure and then these mappings are for the non-canonical aliases), but I think that's just going to result in more mappings to keep track of and I thought it was worse than just using the first-is-canonical convention.

Given that, the capitalization is going to matter. I am game for whatever uppercasing/lowercasing you think is better UX for the spreadsheets. But there's also an argument for keeping it since I assume Preethi chose the current capitalization scheme intentionally.

Reply:
Hm, I see. In that case, I am in support of making the captialization regular (likely every word having a capital first letter?). It doesn't make sense to me that electric panel be "Electric panel" but electric stove be "Electric Stove".

WRT the first alias - I have some concerns that the current groupings (e.g. of "electric stove" and "induction cooktop") will lose valuable data along the way if we're using this list for back-to-csv. Maybe it doesn't matter that much because the description itself will likely include what specifically is being referenced?
(This second comment is not as important for this change)

Reply:
I'll submit this as is, but do the capitalization as an immediate follow-up since it's pretty simple.

Re: your second comment, I agree with you. This is a fundamental problem with our current heat pump setup – we've got 4 kinds of air source heat pump options in spreadsheets and one in JSON. You can collapse 4 to 1 but you can't expand back to 4 once you've lost that detail.

I didn't want to address that here, but ultimately the alias list should be used only for typos/variations of the exact same technology, whereas this is using it also to collapse things that we don't currently have the vocabulary for in the backend. I'll plan to add those as the immediate follow-up as well.